pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -254,6 +259,10 @@ a.link-button {
 ul {
   margin: 1rem;
 
+  li p {
+    @include body-three;
+  }
```

Comment:
Sets the default list paragraph styles

Reply:
I'm curious if it's something we've agreed on or visually selected best default list item text size? Asking to understand if I need to add similar rules to `dashboard` design system

Reply:
This is what it was before for the list element `p`'s. I set the size more explicitly here because some of the list styles were being set to 1rem/16px when I made the scale update. 

Dashboard might have different use cases for list elements than Pegasus, so this might be a designer question.

Reply:
Gotcha, thank you for explaining!